scripts/exglobal_atmos_analysis.sh

```diff
@@ -926,6 +927,8 @@ OBS_INPUT::
    sstviirs       viirs-m     j1          viirs-m_j1          0.0     4     0
    ahibufr        ahi         himawari9   ahi_himawari9       0.0     1     0
    sstviirs       viirs-m     j2          viirs-m_j2          0.0     4     0
+   atmsbufr21     atms        n21         atms_n21            0.0     1     0
+   crisfsbufr21   cris-fsr    n21         cris-fsr_n21        0.0     1     0
```

Comment:
ATMS N21 will be assimilated and the CrIS N21 is a place holder, correct?

Reply:
I can not see the fix files from the github page.  I believe that the use flag for ATMS N21 is set to 1, and the use flags for CrIS-fsr N21 and ABI-g18 are set to -1

I think we already made changes in abias and abias_pc for the three sensors in  v16.3. implementation.

Reply:
@emilyhcliu, I cloned `ADCollard:release/gfsda.v16`.   A check of `fix/global_satinfo.txt` shows that the usage flags for atms_n21, cris-fsr_n21, and abi_g18 are -1 for all channels.

@ADCollard , the PR description indicates that we should see changes in `fix/global_anavinfo.l127.txt`.   This file is identical between `ADCollard:release/gfsda.v16` and `NOAA-EMC:release/gfsda.v16`.   

Should we also see a difference between cris-fsr_n21 weights when comparing `global_satinfo.txt` from `ADCollard:release/gfsda.v16` and `NOAA-EMC:release/gfsda.v16`?

Reply:
Thanks @RussTreadon-NOAA 
@ADCollard did mention that the purpose of this PR is to set CrIS N21, ATMS N21, and ABI G18 to monitoring mode.
So, the use flags for these three sensors should be -1 and they are set correctly in this PR.

Reply:
Yes, both current operations, gfs.v16.3.9, and this PR maintain all channels of atms_n21, cris-fsr_n21, and abi_g18 in monitor mode.   

The only difference between `ADCollard:release/gfsda.v16/fix` and `/lfs/h1/ops/prod/packages/gfs.v16.3.9/fix/fix_gsi` is the addition of `Rcov_crisn21` in `ADCollard:release/gfsda.v16/fix`

Reply:
@RussTreadon-NOAA @emilyhcliu Sorry, I was in a meeting.   Yes, all three instruments need to be in monitor mode to facilitate bias correction spin-up.

Reply:
> 
> @ADCollard , the PR description indicates that we should see changes in `fix/global_anavinfo.l127.txt`. This file is identical between `ADCollard:release/gfsda.v16` and `NOAA-EMC:release/gfsda.v16`.
> 
> Should we also see a difference between cris-fsr_n21 weights when comparing `global_satinfo.txt` from `ADCollard:release/gfsda.v16` and `NOAA-EMC:release/gfsda.v16`?

@RussTreadon-NOAA If there are no changes for `fix/global_anavinfo.l127.txt` and `fix/global_satinfo.txt`, I must have made a mistake.  Let me double-check. (My github/Gerrit skills need some work)